.circleci/config.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       - run: make lint
       - run: make build
       - run: make test
+      - run: make upstream_test
```

Comment:
Nit: `test_upstream` instead of `upstream_test` to match other targets.

But last time I tried doing this, it wasn't as easy as this PR.

Reply:
What's hard about it?

Reply:
41 tests failing with no artifacts. Looks like fun...